src/plugins/metric_vis/public/metric_vis_controller.js

```diff
@@ -8,13 +9,22 @@ define(function (require) {
 
     var metrics = $scope.metrics = [];
 
+    $scope.isNullorNaN = function (val) {
```

Comment:
Why is this attached to $scope? I don't see it used in any views or child scopes.